examples/configs/neurobooth_os_config.json

```diff
@@ -1,26 +1,28 @@
 {
-    "data_out": "C:\\neurobooth\\neurobooth_data\\\\",
-    "nas": "Z:\\data\\\\",
+    "remote_data_dir": "Z:\\data\\\\",
     "video_tasks": "C:\\Users\\STM\\Dropbox (Partners HealthCare)\\Neurobooth Videos for tasks\\Videos_to_present",
     "cam_inx_lowfeed": 0,
     "default_log_path": "D:\\neurobooth\\neurobooth_logs",
     "acquisition": {
         "name": "acq",
         "user": "ACQ",
         "pass": "",
-	"port": 1280
+	    "port": 1280,
+        "data_out": "D:\\neurobooth\\neurobooth_data\\\\"
```

Comment:
What do you think about "local_data_dir" or "local_data_out"? This is to contrast it with "remote_data_dir" and make it more clear that it's a machine-specific setting.

Reply:
agree